--- nasm-0.98.32/nasm.c	Wed May 15 01:54:46 2002
+++ nasm-0.98.32/nasm.c.org	Tue May 21 11:53:35 2002
@@ -427,13 +427,14 @@
 	    using_debug_info = TRUE;
 	    break;
 	  case 'h':
-	    printf("usage: nasm [-@ response file] [-o outfile] [-f format] "
+	    {const char *help_text =
+	         "usage: nasm [-@ response file] [-o outfile] [-f format] "
 		   "[-l listfile]\n"
 		   "            [options...] [--] filename\n"
 		   "    or nasm -r   for version info (obsolete)\n"
 		   "    or nasm -v   for version info (preferred)\n\n"
 		   "    -t          Assemble in SciTech TASM compatible mode\n"
-		   "    -g          Generate debug information in selected format.\n");
+		   "    -g          Generate debug information in selected format.\n";
 	    printf("    -e          preprocess only (writes output to stdout by default)\n"
 		   "    -a          don't preprocess (assemble only)\n"
 		   "    -M          generate Makefile dependencies on stdout\n\n"
@@ -446,7 +447,8 @@
 		   "    -D<macro>[=<value>] pre-defines a macro\n"
 		   "    -U<macro>   undefines a macro\n"
 		   "    -w+foo      enables warnings about foo; -w-foo disables them\n"
-		   "where foo can be:\n");
+		    "where foo can be:\n");
+		               printf(help_text);}
 	    for (i=1; i<=ERR_WARN_MAX; i++)
 		printf("    %-23s %s (default %s)\n",
 		       suppressed_names[i], suppressed_what[i],
