--- nasm-0.98.32/doc/nasmdoc.src.orig	Fri May 17 17:54:24 2002
+++ nasm-0.98.32/doc/nasmdoc.src	Tue May 21 21:32:06 2002
@@ -2,7 +2,7 @@
 \#
 \# Source code to NASM documentation
 \#
-\M{category}{Programming}
+\M{category}{Programming tools:}
 \M{title}{NASM - The Netwide Assembler}
 \M{year}{2002}
 \M{author}{The NASM Development Team}
--- nasm-0.98.32/doc/rdsrc.pl.orig	Thu May 16 20:15:31 2002
+++ nasm-0.98.32/doc/rdsrc.pl	Tue May 21 21:37:07 2002
@@ -884,7 +884,7 @@
   print "\@setfilename ",$metadata{'infofile'},".info\n";
   print "\@dircategory ",$metadata{'category'},"\n";
   print "\@direntry\n";
-  printf "* %-28s %s.\n",
+  printf "* %-37s %s\n",
   sprintf('%s: (%s).', $metadata{'infoname'}, $metadata{'infofile'}),
   $metadata{'infotitle'};
   print "\@end direntry\n";
